thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/datasource/sql/SqlThirdEyeDataSource.java

```diff
@@ -45,11 +45,13 @@
   private static final ThirdEyeCacheRegistry CACHE_REGISTRY_INSTANCE = ThirdEyeCacheRegistry.getInstance();
   protected LoadingCache<RelationalQuery, ThirdEyeResultSetGroup> sqlResponseCache;
   private SqlResponseCacheLoader sqlResponseCacheLoader;
-  public static final String DATA_SOURCE_NAME = SqlThirdEyeDataSource.class.getSimpleName();
-
+  private String serverTimezone = "UTC";
 
 
   public SqlThirdEyeDataSource(Map<String, Object> properties) throws Exception {
+    if (properties.containsKey("serverTimezone")) {
```

Comment:
So the `serverTimeZone` have to be configured? What about using 'TimeZone tz = TimeZone.getDefault();' It will get the local time zone automatically. 
See https://docs.oracle.com/javase/7/docs/api/java/util/TimeZone.html

Reply:
Good to know! I didn't realize there is a way to get local time zone. I will be using DateTimeZone.getDefault().